helm/reana/README.md

```diff
@@ -99,4 +99,4 @@ This Helm automatically prefixes all names using the release name to avoid colli
 | `quota.disk_update`                                      | Cronjob pattern representing how often the users disk quota usage should be updated. Leave it empty to deactivate it | "0 3 * * *"     |
 | `quota.default_disk_limit`                               | Default users disk quota limit in bytes.                                             | None                                            |
 | `quota.default_cpu_limit`                                | Default users CPU quota limit in milliseconds.                                       | None                                            |
-| `workspaces.paths`                                       | List of additional workspace paths, e.g `{cluster_node_path_1, cluster_node_path_2}`. The first one listed different from `/var/reana` will be the default workspace root path. Any POSIX filesystem mounted on cluster nodes is supported | None               |
\ No newline at end of file
+| `workspaces.paths`                                       | JSON list of additional workspace paths as strings. Each mount string is composed by a key cluster_node_path(path to the directory to be mounted from the Kubernetes nodes) and a cluster_pod_mountpath (path inside the cluster containers where the `cluster_node_path` will be mounted) e.g. `cluster_node_path:cluster_pod_mountpath`. The first value listed different from `shared_volume_path` will be the default workspace root path. Any POSIX filesystem mounted on cluster nodes is supported | None               |
```

Comment:
Minor suggestion:
``
List of additional workspace paths as strings. Each mount string is composed of a key `hostPath` (path to the directory to be mounted from the Kubernetes nodes) and a `mountPath` (path inside the cluster containers where the `hostPath` will be mounted) e.g. `hostPath:mountPath`. Any POSIX filesystem mounted on cluster nodes is supported | None 
``
WDYT?